tests/h/search/query_test.py

```diff
@@ -58,8 +58,8 @@ def test_logged_out_user_can_see_shared_annotations(self, search, Annotation):
 
         assert sorted(result.annotation_ids) == sorted(shared_ids)
 
-    def test_logged_in_user_can_only_see_their_private_annotations(self,
-            search, pyramid_config, Annotation):
+    def test_logged_in_user_can_only_see_their_private_annotations(self, search, pyramid_config,
+                                                                   Annotation):
```

Comment:
This change and the one below are just fixes for indentation in continuation lines highlighted by my editor.